.ci/e2e.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+#####
+# This script is created to install kuberhealthy with a few basic checks in a minikube cluster.
+# In the long run I hope that we can use it to run test cases.
+#####
+
+# Set NS
+NS=kuberhealthy
+name=kuberhealthy
+
+# Create namespace
+kubectl create namespace $NS
+
+# Sometimes the kuberhealthy resources get's created...
+sleep 2
+
+# Create kuberhealthy crd etc.
+kubectl create -f .ci/kuberhealthy-e2e.yaml
+
+kubectl -n $NS get khc
+
+sleep 90
+
+echo "get all \n"
+kubectl -n $NS get all
+echo "get khc  \n"
+kubectl -n $NS get khc
+echo "get khs \n"
+kubectl -n $NS get khs
+
+# Wait for kuberhealthy operator to start
+JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'; until kubectl -n $NS get pods -l app=kuberhealthy -o jsonpath="$JSONPATH" 2>&1 |grep -q "Ready=True"; do sleep 1;echo "waiting for kuberhealthy operator to be available"; kubectl get pods -n $NS; done
+
+echo "get deployment logs \n"
+selfLink=$(kubectl get deployment.apps $name -n $NS -o jsonpath={.metadata.selfLink})
+selector=$(kubectl -n $NS get --raw "${selfLink}/scale" | jq -r .status.selector)
+kubectl logs -n $NS --selector $selector
+
+# Verify that the khc went as they should.
+for i in {1..60}
+do
+    khsCount=$(kubectl get -n $NS khs -o yaml |grep "OK: true" |wc -l)
+    cDeploy=$(kubectl -n $NS get pods -l app=kuberhealthy-check |grep deployment |grep Completed |wc -l)
+    cDNS=$(kubectl -n $NS get pods -l app=kuberhealthy-check |grep dns-status-internal |grep Completed |wc -l)
+    cDS=$(kubectl -n $NS get pods -l app=kuberhealthy-check |grep daemonset |grep Completed |wc -l)
```

Comment:
If we were to use helm -- there'd need to be a way to grep the `enabled` values for each check to verify the checks completed their runs

Reply:
Hmm I haven't used helm in production for a year or something like that but if I remember correctly helm2 didn't care about if the pods actually starts it just cares if it have performed a apply more or less and a helm ls would still show installed.

I don't think this have changed in helm3, if so great.
So I think we would still have to check if the pods is actually finished a run.

Reply:
Ignore this comment ^^ since we're going to be running all checks, we'd just have to check that each one successfully runs!